docs/blog/november-contributions-showcase.md

```diff
@@ -0,0 +1,111 @@
+    - property: og:description
+      content: A showcase of e18e-focused contributions from November
+  - - meta
+    - property: og:image
+      content: https://e18e.dev/og/november-contributions-showcase.png
+---
+
+# November contributions
+
+_December 3, 2024_
+
+Another month, another great list of performance gains across the ecosystem. This month, we've also seen a huge effort to improving visibility of these things so more people can get involved. Let's take a look!
+
+## Libraries
+
+### kasi
+
+A set of utilities we've seen quite often in the wild are case transforms. For example, transforming to and from camel case (e.g. from `foo-bar` to `fooBar`).
+
+Often, this has been achieved by using a micro-utility dependency or by repeating a very similar function to everyone else who needed this logic.
+
+This is a really good example of something which is too big to be copied everywhere, but too small to justify its own package.
+
+So what do we do in these cases? We group the common case transforms and use _that_ as a library.
+
+That is exactly what Fabio has done with [kasi](https://github.com/fabiospampinato/kasi)!
+
+kasi is a great collection of functions for transforming between various cases (pascal, camel, snake, etc). Instead of pulling in many micro-packages, or repeating code, we can now use this.
+
+Great work as always, Fabio! :pray:
+
+### unicode-segmenter
+
+As part of the bluesky contributions mentioned later in this post, some of the community noticed a large chunk of the JS bundle was taken up by [graphemer](https://github.com/flmnt/graphemer).
+
+This library is used to split or count the number of grapheme clusters in a string (i.e. the visual symbols we would perceive as individual characters). It does a good job at this but comes in at a whopping 812KB!
+
+Fortunately, [@hyeseong.kim](https://bsky.app/profile/hyeseong.kim) came along and built [unicode-segmenter](https://github.com/cometkim/unicode-segmenter) :tada: This library achieves the same functionality at a fraction of the size (250KB).
+
+Really great work here. This helped us reduce bundle sizes in a whole bunch of projects.
+
+### sonda
+
+As part of the cleanup effort in the e18e space, it is a common need that we want to analyze the bundle of a project. Usually, we do this to find duplicate dependencies, oversized dependencies and so on.
+
+In the wild, there are not many tools to do this analysis and do it accurately. This has left us with often manually inspecting bundles and the process taking a lot longer than it should.
+
+All good, though, [Filip](https://bsky.app/profile/filipsobol.bsky.social) has come to the rescue with his new tool: [sonda](https://github.com/filipsobol/sonda/tree/main/packages/sonda)!
+
+Sonda is a bundle analyzer which works with all popular bundlers by leaning on sourcemaps to determine the bundle contents. This works great and has already helped us with many investigations into footprint reduction of some projects :pray:
```

Comment:
> This [...] already helped us with many investigations into footprint reduction of some projects

Maybe I'm working against myself here, but is it true? :sweat_smile:

Reply:
i've been using it on vite's bundle recently and a few others to catch duplicates etc

I've seen a few others using it too, but its difficult to say for sure what the actual usage is

maybe I should reword that it is _starting_ to help us?

Reply:
Nah, I just wanted to make sure that this is not made up just to make it sound more exciting :wink: